docs/GettingStarted.rst

```diff
@@ -0,0 +1,86 @@
+.. raw:: html
+
+   <!-- This Source Code Form is subject to the terms of the Mozilla Public
+      - License, v. 2.0. If a copy of the MPL was not distributed with this
+      - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+===============
+Getting Started
+===============
+Get started by installing Frost and running some pre-built tests.
+
+Installing Frost
+================
+
+Clone the repo::
+
+   git clone git@github.com:mozilla/frost.git
+
+Create a virtual environment::
+
+   cd frost
+   python3 -m venv venv  
+
+Activate the virtual environment::
+
+   . venv/bin/activate
+
+Upgrade pip::
+
+   pip install --upgrade pip
+
+Install Requirements::
+
+   pip install -r requirements.txt
+
```

Comment:
One thing not in the `Makefile` is installing 'frost' in development mode, so you can use the `frost` command as well. To do that,  run:
```python
pip install -e .
```